src/en/guide/testing/integrationTesting.adoc

```diff
@@ -72,6 +73,11 @@ import spock.lang.*
 @Integration
 @Rollback
 class BookSpec extends Specification {
+    
+    void setup() {
+        // Do not use this method
```

Comment:
These two lines should not be commented out in the example, and the comment should be
`// This line will not roll back`